scalyr_agent/builtin_monitors/syslog_monitor.py

```diff
@@ -42,6 +43,12 @@
 from six.moves import range
 import six.moves.socketserver
 
+try:
+    # Only available for python >= 3.6
+    import syslogmp
```

Comment:
If this lib may be used by agent which is not from our images, then it seems that it has to be bundled here as a third party library. 

luckily we are close to new linux binaries so soon we won't have to do that.

Reply:
Awesome